src/web/core/src/main/java/org/geoserver/web/GeoServerErrorPage.html

```diff
@@ -5,9 +5,7 @@
             <wicket:message key="GeoServerErrorPage.loginRequired"></wicket:message>
         </p>
         <p wicket:id="trace-explanation">
-            <wicket:message key="GeoServerErrorPage.preamble"></wicket:message> 
-            <a wicket:id="jiraLink"></a>
-            <wicket:message key="GeoServerErrorPage.conclusion"></wicket:message> 
+            <a wicket:id="userListLink"></a>
```

Comment:
Can we set `target="_blank"` on the link to request a new browser window? I usually avoid these but in this case it could stop the user from losing the stack trace.